constraint/pkg/client/drivers/k8scel/driver_test.go

```diff
@@ -285,42 +276,13 @@ func TestValidation(t *testing.T) {
 						Message:    "unexpected name",
 					},
 				},
-			}, ptr.To[string](string(VAPDefaultYes))),
-			constraint:         makeConstraint(nil),
+				GenerateVAP: ptr.To[bool](true),
+			}),
+			constraint:         makeConstraint(),
 			isAdmissionRequest: true,
-			vapDefault:         ptr.To[vapDefault](VAPDefaultNo),
+			vapDefault:         false,
 			expectedViolations: false,
 		},
-		{
-			name: "Unsatisfied constraint, default assume no VAP, admission request, constraint override",
-			template: makeTemplateWithSource(&schema.Source{
-				Validations: []schema.Validation{
-					{
-						Expression: `object.metadata.name == "unrecognizable-name"`,
-						Message:    "unexpected name",
-					},
-				},
-			}, nil),
-			constraint:         makeConstraint(ptr.To[string](string(VAPDefaultYes))),
-			isAdmissionRequest: true,
```

Comment:
do we not need to test constraint override cases?

Reply:
Constraint is inheriting the generation behavior now and not overriding.

Reply:
But sounds like `scopedEnforcementActions` spec field on constraints can override? if so, then we should include this in the test?

Reply:
If this above comment makes sense - https://github.com/open-policy-agent/frameworks/pull/427#discussion_r1646515841 - then yes we should include this test also.

Reply:
@ritazh since `scopedEnforcementActions` spec doesn't exists in this PR, would it be best if these tests are included again as part of - #403 - where we are adding the spec to be able to override.